docs/index.md

```diff
@@ -91,30 +73,11 @@ provider "artifactory" {
 }
 ```
 
-### JFrog API Key Header
-Artifactory API keys may be used via the `X-JFrog-Art-Api` header by providing the `api_key` field in the provider block.
-Getting this value from the environment is supported with the `ARTIFACTORY_API_KEY` variable
-
-Usage:
-```hcl
-# Configure the Artifactory provider
-provider "artifactory" {
-  url = "artifactory.site.com/artifactory"
-  api_key = "abc...xy"
-}
-```
-
 ## Argument Reference
 
 The following arguments are supported:
 
-* `url` - (Required) URL of Artifactory. This can also be sourced from the `ARTIFACTORY_URL` environment variable.
-* `username` - (Optional) Username for basic auth. Requires `password` to be set.
-    Conflicts with `api_key`, and `access_token`. This can also be sourced from the `ARTIFACTORY_USERNAME` environment variable.
-* `password` - (Optional) Password for basic auth. Requires `username` to be set.
-    Conflicts with `api_key`, and `access_token`. This can also be sourced from the `ARTIFACTORY_PASSWORD` environment variable.
-* `api_key` - (Optional) API key for api auth. Uses `X-JFrog-Art-Api` header.
-    Conflicts with `username`, `password`, and `access_token`. This can also be sourced from the `ARTIFACTORY_API_KEY` environment variable.
-* `access_token` - (Optional) API key for token auth. Uses `Authorization: Bearer` header. For xray functionality, this is the only auth method accepted
-    Conflicts with `username` and `password`, and `api_key`. This can also be sourced from the `ARTIFACTORY_ACCESS_TOKEN` environment variable.
+* `url` - (Optional) URL of Artifactory. This can also be sourced from the `ARTIFACTORY_URL` environment variable.
+* `access_token` - (Optional) API key for token auth. Uses `Authorization: Bearer` header. 
```

Comment:
This isn't a "API key", so let's clean up the documentation. I know this is existing documentation but there's no need to mention this token is used in the HTTP Authorization header as Bearer token.

Reply:
removed